pkgs/development/python-modules/ttfautohint-py/default.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  gitUpdater,
+  pytestCheckHook,
+  setuptools,
+  setuptools-scm,
+  distutils,
+  ttfautohint,
+  fonttools,
+}:
+
+buildPythonPackage rec {
+  pname = "ttfautohint-py";
+  version = "0.5.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "fonttools";
+    repo = "ttfautohint-py";
+    rev = "v${version}";
+    hash = "sha256-NTog461RpyHKo/Qpicj3tflehaKj9LlZEN9qeCMM6JQ=";
+  };
+
+  postPatch = ''
+    substituteInPlace src/python/ttfautohint/__init__.py \
+      --replace-fail 'find_library("ttfautohint")' '"${lib.getLib ttfautohint}/lib/libttfautohint.so"'
+  '';
+
+  env.TTFAUTOHINTPY_BUNDLE_DLL = false;
+
+  build-system = [
+    setuptools
+    setuptools-scm
+    distutils
+  ];
+
+  buildInputs = [ ttfautohint ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    fonttools
+  ];
+
+  pythonImportsCheck = [ "ttfautohint" ];
+
+  passthru.updateScript = gitUpdater { rev-prefix = "v"; };
```

Comment:
Most likely not needed as the bot and `nix-update` works fine for Python modules.

Reply:
I like being more explicit about updater scripts.

Reply:
done